components/src/core/Drawer/DrawerNavGroup.tsx

```diff
@@ -407,25 +408,37 @@ export const DrawerNavGroup: React.FC<DrawerNavGroupProps> = (props) => {
 
 DrawerNavGroup.displayName = 'DrawerNavGroup';
 
+const NestedNavItemPropTypes = {
+    chevron: PropTypes.bool,
+    collapseIcon: PropTypes.element,
+    divider: PropTypes.bool,
+    expandIcon: PropTypes.element,
+    itemID: PropTypes.string.isRequired,
+    onClick: PropTypes.func,
+    rightComponent: PropTypes.element,
+    statusColor: PropTypes.string,
+    subtitle: PropTypes.string,
+    title: PropTypes.string.isRequired,
+};
+
 DrawerNavGroup.propTypes = {
     activeBackgroundColor: PropTypes.string,
     activeFontColor: PropTypes.string,
     activeIconColor: PropTypes.string,
+    activeItem: PropTypes.string,
+    activeBackgroundShape: PropTypes.oneOf(['round', 'square']),
     backgroundColor: PropTypes.string,
     chevron: PropTypes.bool,
-    titleContent: PropTypes.element,
+    titleContent: PropTypes.node,
```

Comment:
Does this work correctly if you pass in a node instead of an element?

Reply:
titleContent has been a ReactNode type when it's declared in type DrawerNavGroupProps. But I tried in the showcase to pass in a `<></>`, and it works, so I think this change is legit.